cypress/integration/smokeTests/sharedTests/basic-enduser.ts

```diff
@@ -148,6 +148,90 @@ describe('Check external links', () => {
   });
 });
 
+const galaxyWorkflowTuples = [
+  [
+    'github.com/iwc-workflows/sars-cov-2-pe-illumina-wgs-variant-calling/COVID-19-PE-WGS-ILLUMINA',
+    'main',
+    'v0.2.2',
+    'github.com/iwc-workflows/sars-cov-2-pe-illumina-wgs-variant-calling/COVID-19-PE-WGS-ILLUMINA',
+    'Galaxy',
+  ],
```

Comment:
Similar to other comment, can add this to `workflowVersionTuples` so `testWorkflows` handles it